bindings/python/pybind11.hpp

```diff
@@ -0,0 +1,109 @@
+#include <pybind11/pybind11.h>
+
+#include <pinocchio/multibody/model.hpp>
+
+namespace pinocchio {
+namespace cpp2pybind11 {
+namespace bp = boost::python;
+
+template<typename T> pybind11::object to(T& t)
+{
+  // Create PyObject using boost Python
+  typename bp::reference_existing_object::apply<T*>::type converter;
+  PyObject* pyobj = converter(t);
+  // Create the Pybind11 object
+  return pybind11::reinterpret_borrow<pybind11::object>(pyobj);
+}
+template <typename T>
+auto to(T* t) {
```

Comment:
Could you remove auto and use standard C++98 rule?

Reply:
With pybind11, the standard is 11 at least.

Reply:
OK.